git_hooks/README.md

```diff
@@ -0,0 +1,14 @@
+README for git hooks script
+===========================
+git has a way to run scripts, which are invoked by specific git commands.
+The git hooks are located in <root>.git/hooks, and as such are not under version control
```

Comment:
Style: perhaps enclose `<root>.git/hooks` with backticks (i.e. "\`")? Also, probably good to clarify that `root` probably means the mbed TLS project root directory.
Is there a `/` missing between "<root>" and the rest of the path?